draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -120,35 +121,35 @@
     <section title="Introduction" anchor="intro">
       <t>
         The Border Gateway Protocol (BGP) as originally designed is known to be vulnerable to prefix (or route) hijacks and BGP route leaks <xref target="RFC7908"/>.
-        Some existing BGP extensions are able to partially solve these problems.
+        Some existing BGP extensions can partially solve these problems.
         For example, Resource Public Key Infrastructure (RPKI) based route origin validation (RPKI-ROV) <xref target="RFC6480"/> <xref target="RFC6482"/> <xref target="RFC6811"/> <xref target="RFC9319"/> can be used to detect and filter accidental mis-originations.
         <xref target="RFC9234"/> or <xref target="I-D.ietf-grow-route-leak-detection-mitigation"/> can be used to detect and mitigate accidental route leaks.
         While RPKI-ROV can prevent accidental prefix hijacks, malicious forged-origin prefix hijacks can still occur <xref target="RFC9319"/>.
         RFC9319 includes some recommendations for reducing the attack surface for forged-origin prefix hijacks.
       </t>
       <t>
         This document describes procedures that make use of Autonomous System Provider Authorization (ASPA) objects <xref target="I-D.ietf-sidrops-aspa-profile"/> in the RPKI to verify properties of the BGP AS_PATH attribute of advertised routes.
-        ASPA-based AS_PATH verification provides detection and mitigation of route leaks and improbable AS paths.
-        It also provides protection, to some degree, against prefix hijacks with forged-origin or forged-path-segment (<xref target="use-cases"/>).
-        These new ASPA-based procedures automatically detect such anomalous AS_PATHs in announcements that are advertised between AS.
+        ASPA-based AS_PATH verification provides detection and mitigation of route leaks.
+        It also provides protection, to some degree, against prefix hijacks with forged-origin or forged-path-segment (<xref target="property"/>).
+        These new ASPA-based procedures automatically detect such anomalous AS_PATHs in BGP routes that are advertised between ASes.
```

Comment:
if you want to be precise is hould BGP Updates

Reply:
@mitradir 
Yes, correct. Done.